### PR TITLE
Changing IP to hostname in all services, all references.

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

File: cadvisor-dns.service [hunk 1]
```diff
@@ -6,7 +6,7 @@ After=cadvisor.service
 
 [Service]
 ExecStart=/bin/sh -c "\
-  export HOST_IP=$(/usr/bin/ifconfig eth0 | sed -n '2 p' | awk '{print $2}' | cut -d':' -f2); \
+  export HOST_IP=$(hostname); \
```
Comment (catalin-ilea) [inline cadvisor-dns.service]: see @Zapadlo's comment

Comment (catalin-ilea) [inline cadvisor-dns.service]: you can use $HOSTNAME

**Comment by catalin-ilea** [comment]
+1

File: cadvisor-dns.service [hunk 2]
```diff
@@ -6,7 +6,7 @@ After=cadvisor.service
 
 [Service]
 ExecStart=/bin/sh -c "\
-  export HOST_IP=$(/usr/bin/ifconfig eth0 | sed -n '2 p' | awk '{print $2}' | cut -d':' -f2); \
+  export HOST_IP=$HOSTNAME; \
```
Comment (Zapadlo) [inline cadvisor-dns.service]: Sorry to be pedantic, but you really don't need to reassign this variable to $HOST_IP, just use it as the parameter...

Comment (kovacshuni) [inline cadvisor-dns.service]: And everywhere. Fixed.

**Comment by Zapadlo** [comment]
+1

Thanks H!

**Pull request merged**